docs/querying/sql-functions.md

```diff
@@ -1282,22 +1282,75 @@ Returns the following:
 
 ## BLOOM_FILTER
 
-Computes a Bloom filter from values produced by the specified expression.
+Computes a [Bloom filter](../development/extensions-core/bloom-filter.md) from values provided in an expression.
 
-* **Syntax**: `BLOOM_FILTER(expr, <NUMERIC>)`
+
+* **Syntax:** `BLOOM_FILTER(expr, numEntries)`  
+  `numEntries` specifies the maximum number of distinct values before the false positive rate increases.
 * **Function type:** Aggregation
 
+<details><summary>Example</summary>
+
+The following example returns a Base64-encoded Bloom filter string for entries in `agent_category`:
+
+```sql
+SELECT
+  agent_category,
+  BLOOM_FILTER(agent_category, 10) as bloom
+FROM "kttm"
+  GROUP BY agent_category
+```
+
+Returns the following:
+
+| `agent_keys` | `bloom` |
+| -- | -- |
+| `empty` | `"BAAAAAgAAAAAABAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEABAAAAAA"` |
+| `Game console` | `"BAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAQAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgBAAAAAAAAAAAAAAAA"` |
+| `Personal computer` | `"BAAAAAgAAAAAAEAAAAAAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAA"` |
+| `Smart TV` | `"BAAAAAgAAAAAAAAAAAAAgAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgAA"` |
+| `Smartphone` | `"BAAAAAgAAACAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAIAAAAAAAAAAAAAAAAAAIAAAAAAAAAAAAAAAAAAAAA"` |
+| `Tablet` | `"BAAAAAgAAAAAAAAAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAgAAAAAAAAAAAAAAAAAAAAACAAAAAAAAAAAAAAAAAAAAAAIA"` |
+
+</details>
+
 [Learn more](sql-aggregations.md)
 
 ## BLOOM_FILTER_TEST
 
-Returns true if the expression is contained in a Base64-serialized Bloom filter.
+Returns true if an expression is contained in a Base64-encoded [Bloom filter](../development/extensions-core/bloom-filter.md) string.
 
-* **Syntax**: `BLOOM_FILTER_TEST(expr, <STRING>)`
+* **Syntax:** `BLOOM_FILTER_TEST(expr, <STRING>)`
 * **Function type:** Scalar, other
 
+<details><summary>Example</summary>
+
+The following example returns `true` for the Bloom filter string associated with `agent_filter` entry `Game console`:
```

Comment:
This description doesn't describe what the example does. I suggest simplyfying as follows:

The following example checks for the existence of the agent category `Rotary phone` in the specified bloom filter.

```sql
SELECT BLOOM_FILTER_TEST('Rotary phone', 'BAAAAAgAAACAAFAAQAAAgAIAAkIAAAAAQAgAAAAAAAAAAgAAAAIIAAAAAgAAAAAAAAACAAIAAAgBAQAAAAAEABAAAgIA'
```

Returns the following:

|`bloom_test`|
| --- |
|`false`|

The following example checks for the existence of the agent category `Personal computer` in the specified bloom filter:

```sql
SELECT BLOOM_FILTER_TEST('Personal computer', 'BAAAAAgAAACAAFAAQAAAgAIAAkIAAAAAQAgAAAAAAAAAAgAAAAIIAAAAAgAAAAAAAAACAAIAAAgBAQAAAAAEABAAAgIA') AS "bloom_test"
```

Returns the following:

|`bloom_test`|
| --- |
|`true`|